clickhouse_driver/connection.py

```diff
@@ -21,7 +21,7 @@
 from .settings.writer import write_settings
 from .writer import write_varint, write_binary_str
 
-
+logging.basicConfig()
```

Comment:
Why we need `basicConfig` here? Application should care about logging configuration by itself.